CODEOWNERS

```diff
@@ -15,6 +15,7 @@ Makefile*                                                        @istio/wg-test-
 /manifests/                                                      @istio/wg-environments-maintainers
 /operator/                                                       @istio/wg-environments-maintainers
 /pilot/                                                          @istio/wg-networking-maintainers
+/pilot/cmd/pilot-agent/                                          @istio/wg-networking-maintainers @istio/wg-security-maintainers
```

Comment:
Not sure what is the semantic of this file - but changes must have approval from at least one networking and env lead, this is super critical for both. I don't think it should have security related code - just setup.

Reply:
The main chunk of istio agent code is SDS agent and STS server logic. This file contains all the configurations for SDS and STS agents.

Reply:
The semantics is an OR. I would not feel comfortable having changes reviewed only by security team for this folder. SDS and STS - absolutely. But those are not part of this folder. See:

```
pilot/cmd/pilot-agent
├── config.go
├── config_test.go
├── main.go
├── main_test.go
├── request.go
├── status
│   ├── metrics.go
│   ├── ready
│   │   ├── probe.go
│   │   └── probe_test.go
│   ├── server.go
│   ├── server_test.go
│   ├── test-cert
│   │   ├── cert.crt
│   │   └── cert.key
│   └── util
│       ├── stats.go
│       └── util.go
└── wait.go
```

The SDS code is all under security/ already.

Reply:
It's the place where all the security configuration are passed to Istio agent (see main.go). Currently this file is only controlled by networking maintainers. I think we need to set up some shared ownership for this kind of shared files.

Reply:
There is ~10 lines of code passing parameters to security code, then 2.5k lines of networking code. I don't see why this requires co-ownership. Its not like networking owners are slow to review. Is there some more context on why you feel this is required?

Reply:
ok, I removed the update on this directory.